dev/instantsearch.css

```diff
@@ -1 +0,0 @@
-../dist/instantsearch.css
```

Comment:
I'm thinking about the `staging` command:
```bash
mv ./dev/instantsearch.css ./dev/instantsearch.css.symlink
cp ./dist/instantsearch.css ./dev/instantsearch.css

surge dev

rm ./dev/instantsearch.css
mv ./dev/instantsearch.css.symlink ./dev/instantsearch.css
rm ./dev/bundle.js
rm ./dev/bundle.js.map
```

Is there an easy way to revert this?